package.json

```diff
@@ -24,6 +24,9 @@
     "webpack-dev-server": "^3.11.2"
   },
   "dependencies": {
+    "libxmljs": "^0.19.10",
```

Comment:
None of these libraries is needed

Same with changes to `package-lock.json`

Reply:
I needed to use libxml and node-gyp, because after yarn start I was getting binding errors.

Reply:
@KonradLinkowski

Reply:
Weird I don't get errors like this without the libraries

Reply:
ok